linkedlistutil.cpp

```diff
@@ -0,0 +1,173 @@
+#include <iostream>
+#include <cstdlib>
+#include <stack>
+#include <cmath>
+#include <vector>
+#include <cstring>
+using namespace std;
+
+
+
+typedef struct node
+{
+	int data;
+	struct node * next;
+	node(int x) {
```

Comment:
its actually redundant. I thought of adding a constructor. but i did not knw exactly how to implement it. so if you see below there is absolutely no use of constructor in this.